src/js/main.js

```diff
@@ -1,5 +1,5 @@
 import { productList } from "./productList.mjs";
 import { loadHeaderFooter } from "./utils.mjs";
 
-productList(".product-list", "tents");
+// productList(".product-list", "tents");
```

Comment:
This caused a console error because there was no such element with the `product-list` class on this page.